akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -203,8 +203,8 @@ object HttpEntity {
    *
    * If the given `chunkSize` is -1 the default chunk size is used.
    */
-  def apply(contentType: ContentType, file: File, chunkSize: Int = -1): UniversalEntity = {
-    val fileLength = file.length
+  def apply(contentType: ContentType, file: Path, chunkSize: Int = -1): UniversalEntity = {
```

Comment:
I like `fromPath` but @ktoso have a better overview of the naming of other things here